src/model/dto/user-profile-dto.ts

```diff
@@ -4,8 +4,8 @@ export class UserProfile {
     lastName!: string
     email!: string
     phone!: any
-    emailVerified!: boolean
-    username!: string
+    // emailVerified!: boolean
```

Comment:
why are these fields taken out?

Reply:
Unwanted . now restored , upto consumer to use or not